### PR TITLE
Added ability to search users.... and restored testing coverage %

### LINKED ISSUE #86: Add ability to search candidates

### PR DESCRIPTION
**Pull request by jackofseattle**
This is a smaller PR than it looks like.  Here's the gist:

Added in ability to search users into the toolbar.
Refactored the toolbar so we get easier styling.
Enabled the test coverage percentage checks.
Expanded tests until the test coverage was sufficient.

Fixes #86

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -13,13 +13,15 @@
...
       <md-toolbar class="md-whiteframe-3dp" id="main-header">
-        <div layout="row" layout-padding class="header-headline" layout-alignment="start center">
-          <div flex-offset="10" class="md-headline align-headline">
+        <div class="md-toolbar-tools">
+          <h2>
             <a href="#/" id="logo">InRhythm - Vivace</a>
-          </div>
+          </h2>
         </div>
+
```
Comment (jackofseattle) [inline client/index.html]: Seems I forgot to reapply the search changes. Updating now

**Comment by jshor** [comment]
LGTM

**Comment by daviskoh** [comment]
merge conflicts

**Comment by jackofseattle** [comment]
@daviskoh merge conflicts resolved

File: client/vivace/comments/controller.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ module.exports = [
   '$routeParams',
   function(DataModel, $mdMedia, $mdDialog, $routeParams) {
 
+    /* istanbul ignore next */
```
Comment (daviskoh) [inline client/vivace/comments/controller.js]: Just curious, why are we having istanbul ignore these various lines in code cov reports?

Comment (jackofseattle) [inline client/vivace/comments/controller.js]: The $mdDialog form is rather hard for us to test - it comes with tests from the ng-material suite though so I'm not as interested in having to test it from our end.

I don't want these lines to negative effect our coverage report, so I'm just marking them as irrelevant

**Comment by daviskoh** [comment]
:+1:

**Pull request merged**